Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.8
 
-MAINTAINER Petr Hanak  <petr.hanak@sap.com>
+LABEL key="Petr Hanak  <petr.hanak@sap.com>"
```

Comment:
s/key/MAINTAINER/

Reply:
Yeah, should be `LABEL maintainer="petr.hanak@sap.com"` I think

Reply:
Yep, my point was to replace key with maitainer in your commit (I should not have used capitals).